src/restart_H5.F90

```diff
@@ -1109,7 +1126,8 @@ subroutine restart_init_h5
      
      rank=2
      dims2(1) = n1
-     dims2(2) = 4
+     ! with turbulence n_cons is always 6
+     dims2(2) = 6
```

Comment:
is this right?  M1_matter_source is still only 4 big, not 6

Reply:
Good point. I guess that it does not give you an error since maybe as long as the given dimension is larger than the actual dimension of the array that needs to be read it's fine. I think I changed this because otherwise line 1132 would cause the code to stop, and I didn't know how important that line was. I could change it back to dims2(2) = 4 and then delete 1132. Is there any case where line 1132 could be a helpful check?

Reply:
right, ok, 1132 can go.  This was incase the M1 source terms were used inside the hydro step.  But this is not done anymore, M1_matter_source is purely in the transport.

Reply:
sounds good. Done!